shotover-proxy/src/transforms/debug/force_parse.rs

```diff
@@ -10,17 +10,40 @@ use anyhow::Result;
 use async_trait::async_trait;
 use serde::Deserialize;
 
+/// Messages that pass through this transform will be parsed.
+/// Must be individually enabled at the request or response level.
 #[derive(Deserialize, Debug, Clone)]
 pub struct DebugForceParseConfig {
-    parse_requests: Option<bool>,
-    parse_responses: Option<bool>,
+    parse_requests: bool,
+    parse_responses: bool,
 }
 
 impl DebugForceParseConfig {
     pub async fn get_transform(&self) -> Result<Transforms> {
         Ok(Transforms::DebugForceParse(DebugForceParse {
-            parse_requests: self.parse_requests.unwrap_or(true),
-            parse_responses: self.parse_responses.unwrap_or(true),
+            parse_requests: self.parse_requests,
```

Comment:
Is there a reason for the configuration options?

I'm thinking to remove all this config, have a transform called DebugForceCodec and just test it all. 

I can't think of a situation where we would want to this separately so might as well keep it simple.

Reply:
Only to match DebugForceParse, which does make use of the config for benchmarking purposes.
I have considered removing the config from DebugForceParse though and I'm happy to do that if you think it makes sense.